tests/VMelnalksnis.NordigenDotNet.Tests.Integration/Accounts/AccountClientTests.cs

```diff
@@ -60,16 +60,19 @@ public async Task GetDetails_ShouldReturnExpected()
 	{
 		var accountDetails = await _nordigenClient.Accounts.GetDetails(_accountId);
 
-		accountDetails.Should().BeEquivalentTo(new AccountDetails
+		accountDetails.Should().BeEquivalentTo(
```

Comment:
If using multiple assertions, should add an `AssertionScope`:
```suggestion
        using var scope = new AssertionScope();
        
		accountDetails.Should().BeEquivalentTo(
```

Reply:
Updated this